### PR TITLE
Feature/rkm66 72 e ticket new

### PR DESCRIPTION
**Pull request by vinitian**
## Change made

-   [ ] Bug fixes
-   [x] New features
-   [ ] Breaking changes

## Describe what you have done

-

### New Features

- new e-ticket page

### Fix

- 

### Others

-

### CODE REVIEW

File: src/pages/e-ticket.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+const ETicket = () => {
+    return (
+        <div className="min-h-screen">
+            <div>
+                <Link href="https://freshmen2023.sgcu.in.th/profile">
+                    <button className="justify-left relative h-10 w-10 rounded-lg px-16 py-16">
+                        <div className="">
+                            <Image
+                                //change icon to arrow-uturn-left in heroicons.com
+                                src={'/images/icons8-instagram.svg'}
```
Comment (leomotors) [inline src/pages/e-ticket.tsx:16]:
```suggestion
                                src="/images/icons8-instagram.svg"
```

File: src/pages/e-ticket.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+// using https://github.com/dephraiim/qrcode
+// example: https://qr-service.vercel.app/api/code?text=https://freshmen2023.sgcu.in.th/&size=200
+
+const ETicket = () => {
+    return (
+        <div className="min-h-screen">
+            <div>
+                <Link href="https://freshmen2023.sgcu.in.th/profile">
+                    <button className="justify-left relative h-10 w-10 rounded-lg px-16 py-16">
```
Comment (boomchanotai) [inline src/pages/e-ticket.tsx:12]: dont have "justify-left" class in tailwindcss

File: src/pages/e-ticket.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                        <div className="">
```
Comment (boomchanotai) [inline src/pages/e-ticket.tsx:13]: line 13 : no need this div

File: src/pages/e-ticket.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+                    </button>
+                </Link>
+            </div>
+            <div className="flex w-full items-center justify-center">
+                <div className="mx-8 my-24 rounded-3xl bg-white px-6 py-8">
+                    <img
+                        alt="E-ticket QR Code"
+                        //please change the QR Code
+                        src="https://qr-service.vercel.app/api/code?text=https://freshmen2023.sgcu.in.th/&size=300"
+                    />
```
Comment (boomchanotai) [inline src/pages/e-ticket.tsx:30]: please use Image tag from next/image
and you have to register domain in next.config.js

**Review by boomchanotai — COMMENTED** [review]
a little bit comment plz read it

File: src/pages/e-ticket.tsx (hunk 3 above)
Comment (shalluv) [inline src/pages/e-ticket.tsx:13]: This `div` is not necessary.

File: src/pages/e-ticket.tsx (hunk 2 above)
Comment (shalluv) [inline src/pages/e-ticket.tsx:12]: `px-16` and `py-16` can be simply written as `p-16`

File: src/pages/e-ticket.tsx (hunk 3 above)
Comment (shalluv) [reply src/pages/e-ticket.tsx:13]: What I meant is you don't need to place the image within this div. Doing so only overcomplicates the code.

File: src/pages/e-ticket.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
+import Image from 'next/image';
+import Link from 'next/link';
+
+// using https://github.com/dephraiim/qrcode
+// example: https://qr-service.vercel.app/api/code?text=https://freshmen2023.sgcu.in.th/&size=200
+
+const ETicket = () => {
+    return (
+        <div className="min-h-screen">
+            <div>
```
Comment (shalluv) [inline src/pages/e-ticket.tsx:10]: You don't need to place `Link` within this div.

File: src/pages/e-ticket.tsx [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+                                alt="Go back"
+                                fill
+                            />
+                        </div>
+                    </button>
+                </Link>
+            </div>
+            <div className="flex w-full items-center justify-center">
+                <div className="mx-8 my-24 rounded-3xl bg-white px-6 py-8">
+                    <img
```
Comment (shalluv) [inline src/pages/e-ticket.tsx:26]: It would be better to utilize the `Image` feature available within `Next.JS`.

**Review by shalluv — CHANGES_REQUESTED** [review]
Nice work! Just a few more changes are needed.

File: src/pages/e-ticket.tsx (hunk 2 above)
Comment (leomotors) [reply src/pages/e-ticket.tsx:12]: Please avoid using `relative` and use flex box to align them instead

**Review by shalluv — APPROVED** [review]
Nice!

**Pull request merged**